pkg/runtime/wfengine/wfengine_test.go

```diff
@@ -162,16 +164,132 @@ func TestSingleActivityWorkflow(t *testing.T) {
 
 	ctx := context.Background()
 	client, engine := startEngine(ctx, t, r)
+
 	for _, opt := range GetTestOptions() {
+		suffix := opt(engine)
 		t.Run(opt(engine), func(t *testing.T) {
```

Comment:
Should we be using `suffix` here?
```suggestion
		t.Run(suffix, func(t *testing.T) {
```